packages/@o3r/core/schematics/ng-add/project-setup/index.ts

```diff
@@ -51,7 +51,8 @@ export const prepareProject = (options: NgAddSchematicsSchema) => async (tree: T
     ...(options.enableLocalization ? ['@o3r/localization'] : []),
     ...(options.enableCustomization ? ['@o3r/components', '@o3r/configuration'] : []),
     ...(options.enableStorybook ? ['@o3r/storybook'] : []),
-    ...(installOtterLinter ? ['@o3r/eslint-config-otter'] : [])
+    ...(installOtterLinter ? ['@o3r/eslint-config-otter'] : []),
+    ...(projectType ? ['@ama-sdk/core'] : ['@ama-sdk/schematics'])
```

Comment:
should this really comes with `@o3r/core` ?
I think this should not be installed by default, maybe it can be included in a preset ?

Reply:
updated. as discussed I called the helper function to retrieve the peer deps instead of hardcoding them.